api_server/src/main/java/io/zetch/app/domain/Restaurant.java

```diff
@@ -0,0 +1,87 @@
+/**
+ * Restaurant entity definition
+ */
+package io.zetch.app.domain;
+
+import javax.persistence.*;
+
+@Entity
+public class Restaurant {
+
+    @Id
+    @GeneratedValue
+    private Integer id;
+
+    @OneToOne  // A restaurant can only have one owner
```

Comment:
I'd suggest making it scalable by changing it to `OneToMany` as we can possibly consider multiple managers manage a restaurant in the future.